connector-drawer/src/test/java/com/redhat/cloud/notifications/connector/drawer/DrawerConnectorRoutesTest.java

```diff
@@ -74,6 +74,9 @@ protected void testRedeliveredNotification() { }
     @Disabled(value = "Not applicable on drawer use case")
     protected void testSuccessfulNotification() { }
 
+    @Disabled(value = "Not applicable on drawer use case")
+    protected void testSendReinjectionRoute() { }
```

Comment:
You are right, yes. Hmmm... I am not sure what to do. We could merge it so that we can benefit from reinjections as you suggested. I hope I don't forget about the test, though :laughing:

Reply:
Sure :)